dll/cpl/desk/desktop.c

```diff
@@ -7,7 +7,19 @@
 
 #include "desk.h"
 
-#define FCIDM_DESKBROWSER_REFRESH    0xA220  /* From shresdef.h */
+#include <shlwapi.h>
+#include <shellapi.h>
+
+/* From shresdef.h */
+#define FCIDM_DESKBROWSER_REFRESH    0xA220
+
+#define IDS_TITLE_MYCOMP  30386
+#define IDS_TITLE_MYNET   30387
+#define IDS_TITLE_BIN_1   30388
+#define IDS_TITLE_BIN_0   30389
+
+/* Workaround */
```

Comment:
Workaround for? (should be mentioned in the comment)

Reply:
Added comment.